### ISSUE TITLE
Wildcards in .swiftlint config are ignored.

### ISSUE THREAD

**Issue by callummccoll** [vknabel/vscode-swiftlint#51]
I am adding some paths to the excluded rule in my swiftlint config to ignore some test files:

```
excluded:
  - Tests/*/*Tests.swift
  - Tests/*/*/*Tests.swift
  - Tests/*/*/*/*Tests.swift
```

However, it seems that the extension is ignoring these paths as I still get swiftlint warnings in vscode for these excluded files. If I run swiftlint from the command line, it ignores these excluded files and everything is fine.

I can't just ignore the Tests folder as I have files (that don't end in "Test.swift") that I want to lint.

**Comment by vknabel** [comment]
Hi @callummccoll. Thanks for opening an issue. Can you either share your project or create a reproduction repository which also includes the vscode settings of Swiftlint you have?

**Comment by callummccoll** [comment]
Hey mate, I created a repo that demonstrates the problem:

https://github.com/callummccoll/ExamplePackage

Take a look at Tests/ExamplePackageTests/ExamplePackageTests.swift for the swiftlint warning that is triggered in vscode, but doesn't show up on the command line.

**Comment by vknabel** [comment]
Thanks for the reproduction! It seems like a `swiftlint --config .swiftlint.yml Tests/ExamplePackageTests/ExamplePackageTests.swift` does not respect the underlying `excluded` option. Not sure if this is intended behavior or a bug.

As a temporary workaround if you are not using nested configs, you can manually set `"swiftlint.configSearchPaths": [".swiftlint.yml"]`. This will explicitly disable the nested configs, but allows the extension to filter out excluded results.

**Comment by vknabel** [comment]
@callummccoll after some quick research I found that this is a bug in SwiftLint itself.
There is a pull request to fix this issue, but it suffers some performance issues. https://github.com/realm/SwiftLint/pull/3534

Therefore I have to admit this is out of scope of this repository.

After reading the PR, there are two workarounds.
1. If you are not using nested configs, set `"swiftlint.configSearchPaths": [".swiftlint.yml"]`. Then the vscode extension will work around this issue.
2. Add every file path you want to exclude to your SwiftLint config - without using a path pattern like `Tests/**/*.swift`

**Comment by callummccoll** [comment]
Fair enough. Thanks for your help. The configSearchPaths config options works for me.
